app/helpers/policies_helper.rb

```diff
@@ -14,9 +14,13 @@ def policy_agreement_summary_without_html(policy_member_distance)
   # Returns things like "voted strongly against", "has never voted on", etc..
   def policy_agreement_summary(policy_member_distance)
     if policy_member_distance.nil?
-      "voted <strong>unknown about</strong>".html_safe
+        "voted <strong>unknown about</strong>".html_safe
```

Comment:
Be careful about accidentally committing stuff you don't mean to, like this superfluous whitespace.